src/bullsAndCowsCount.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+function geBullsAndCows(secret, guess) {
+  let bulls = 0;
+  let cows = 0;
+  const secretArr = secret.split('');
+  const guessArr = guess.split('');
+
+  for (let i = 0; i < secretArr.length; i++) {
+    if (secretArr[i] === guessArr[i]) {
+      bulls++;
+      secretArr[i] = 'X';
+      guessArr[i] = 'X';
+    }
+  }
+
+  for (let i = 0; i < guessArr.length; i++) {
+    if (guessArr[i] !== 'X' && secretArr.includes(guessArr[i])) {
+      cows++;
+
+      const index = secretArr.indexOf(guessArr[i]);
+
+      secretArr[index] = 'X';
+    }
+  }
```

Comment:
It could be done easier. Try to make it the following way:

Inside your `for` loop compare `guessArr[i]` and `secretArr[i]`. If they are equal increment `bulls`. Else, check if `secretArr.includes(guessArr[i])`. If it includes then increment `cows`.

Then return an object of bulls and cows just as you do it.